YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -294,11 +297,16 @@ Repository::Repository(const std::string& path)
 
     if (repo_already_exists)
     {
+        this->include_idb_ = is_file_tracked(get_original_idb_path());
```

Comment:
remove this-> prefix

Reply:
fixed in 1d1e98c73c4b5c6eec04e07bac56ccb00b47b515

Reply:
this-> is used on multiple lines

Reply:
fixed, for real, in af2b97e0a215bedd92fe3bb036af8012f5654736